### ISSUE TITLE
Issue with post purchase display

### ISSUE THREAD

**Issue by thrijith** [laterpay/laterpay-wordpress-plugin#909]
Set a category default price.

Set a post price as category price.

Create a timepass which is applied to all content except for the category we set on the post

Now as a timepass is created, we can disable post-purchase individually.

Delete the category price we set for post in `Pricing` tab

Check post on the frontend.

![wierdpost](https://user-images.githubusercontent.com/13589980/39635822-33dbe612-4fdc-11e8-83ae-3bf167436cd8.png)

**Comment by thrijith** [comment]
this was fixed in [this](https://github.com/laterpay/laterpay-wordpress-plugin/pull/906/commits/7b524c79ce9246c3d41e8ec91b839a3cfc5cfc84)

**Comment by PatelUtkarsh** [comment]
@ericflarsen Let me know what should we do in this case. We can also show warning if timepass is not created with access to all post.

**Comment by thrijith** [comment]
Hi @ericflarsen,

I have some doubts regarding this issue.

If we create a time-pass excluding a category and disable the individual post purchase.

In above case the post with that category is made free, but  if we have a subscription for the said category, then should it display the option to buy that subscription? Is this is a valid use case? Because we can only disable individual post purchase when there is a time-pass, should that be applied to subscription as well?

In other words post should be free only when there is individual purchase disabled and no time pass or subscription belongs to that post.

**Comment by thrijith** [comment]
Hey @ericflarsen,

1. Till now setting global default as `0` made all posts free, to match cases given in above comment we have made changes accordingly.

2. To disable individual post purchase, it was required to create at least one time pass, now that applies to subscription too, now user would have to create either a timepass or a subscription.

**Comment by ksmoore125** [comment]
I am having difficulty understanding the use case for turning off individual purchases. Do we have a way to tell if anyone has done this? If not, could we implement something easily to gather this information before we put forth effort to fix this ticket?

**Comment by thrijith** [comment]
Hi @ksmoore125,

![screen shot 2018-08-23 at 11 19 54 pm](https://user-images.githubusercontent.com/13589980/44545471-c6c80200-a732-11e8-9d82-852a300d780e.png)

This option allows a user to enable/disable individual post purchase.

For doing that user had to create at least one time pass.
This condition has been extended, now a user has to create either a time pass or subscription to disable it.

>Do we have a way to tell if anyone has done this?

We can get that information from current value of the above given option.

Hope this hepls.

Please let me know if you have any further questions.

**Comment by ksmoore125** [comment]
Let me attempt to rephrase my question: Can we get a list or a count of all WordPress users who have this toggle set to "Posts cannot be purchased individually." ?

**Comment by thrijith** [comment]
Hey @ksmoore125, 

If everything is clear can we proceed with this? As all cases given by Eric are working in PR #935

**Comment by ksmoore125** [comment]
Yes, please move forward. Thanks!
